ast/capabilities.go

```diff
@@ -63,3 +68,52 @@ func LoadCapabilitiesJSON(r io.Reader) (*Capabilities, error) {
 	var c Capabilities
 	return &c, d.Decode(&c)
 }
+
+// LoadCapabilitiesVersion loads a JSON serialized capabilities structure from the specific version.
+func LoadCapabilitiesVersion(version string) (*Capabilities, error) {
+	cvs, err := LoadCapabilitiesVersions()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, cv := range cvs {
+		if cv == version {
+			cont, err := caps.FS.ReadFile(cv + ".json")
+			if err != nil {
+				return nil, err
+			}
+
+			c, err := LoadCapabilitiesJSON(bytes.NewReader(cont))
+			if err != nil {
+				return nil, err
+			}
+			return c, nil
```

Comment:
[nit] this would be equivalent to
```suggestion
			return LoadCapabilitiesJSON(bytes.NewReader(cont))
```

Reply:
db702a974c07068fd71051f11d8f659d15a6c46d